queries.sql

```diff
@@ -16,3 +16,44 @@ SELECT * FROM animals WHERE neutered='t';
 SELECT * FROM animals WHERE name!='Gabumon';
 /*Find all animals with a weight between 10.4kg and 17.3kg (including the animals with the weights that equals precisely 10.4kg or 17.3kg)*/
 SELECT * FROM animals WHERE weight_kg BETWEEN 10.4 AND 17.3;
+
+
+/* New queries */
+BEGIN;
+UPDATE animals SET species= 'unspecified';
+ROLLBACK;
+SELECT * FROM animals;
+/* */
+BEGIN;
+UPDATE animals SET species= 'digimon' where name LIKE '%mon';
+UPDATE animals SET species= 'pokemon' WHERE species IS null;
+COMMIT;
```

Comment:
- [ ] Again here kindly add a select statement to check that changes where made to the database before you commit

Reply:
well resolved! ✔️